dwi_ml/experiment/monitoring.py

```diff
@@ -1,10 +1,12 @@
+# -*- coding: utf-8 -*-
 import numpy as np
 
 
+# Checked!
 class LossHistoryMonitor(object):
```

Comment:
I set it up in my code as a `ValueMonitor`, since we can use it to monitor any kind of metric while training

Reply:
ok